app.js

```diff
@@ -42,6 +43,7 @@ const init = async () => {
     context: ({ req }) => ({
       isUserRequest: true,
       egoToken: (req.headers.authorization || '').split('Bearer ').join(''),
+      Authorization: `Bearer ${(req.headers.authorization || '').split('Bearer ').join('')}` || '',
```

Comment:
Does this line do `Authorization: ${req.headers.authorization || ''}`

Reply:
almost... it ensures that the output is `Bearer {token}` whether the incoming header had the bearer prefix or not.

Reply:
For easier readable, I suggest `Bearer ${(req.headers.authorization || '').replace('Bearer ', '')`

Reply:
that is cleaner, will do

Reply:
FYI, I just realized trimStart could be used. https://lodash.com/docs/4.17.15#trimStart. That's not a big deal

Reply:
accomplished the same with regex ;)
`.replace(/^Bearer[\s]*/, '')`